68.md

```diff
@@ -90,3 +90,5 @@ Only the following media types are accepted:
 - `image/webp`: Web Picture format (WEBP)
 
 Picture events might be used with [NIP-71](71.md)'s kind `34236` to display short vertical videos in the same feed.
+
+Replies to NIP-68 kind `20` feeds should follow [NIP-22 comment](https://github.com/nostr-protocol/nips/blob/master/22.md).
```

Comment:
```suggestion
Replies to NIP-68 kind `20` events should follow [NIP-22 comment](https://github.com/nostr-protocol/nips/blob/master/22.md).
```

Makes more sense to say events and than feeds, since it's the individual events that are being replied to